hack/scripts/ca_metrics_parser.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''
+This script parses metrics from Cluster Autoscaler e2e tests.
+'''
+from __future__ import division
+from __future__ import print_function
+import argparse
+import json
+import sys
+
+
+class CAMetric:
```

Comment:
I know it doesn't matter here, but old-style classes are evil and should be eradicated. Therefore 
`class CAMetric(object):`